### PR TITLE
refactor inputs a bit, and split up address a bit

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

**Comment by ScottDowne** [comment]
Still a bit of a WIP

**Comment by ScottDowne** [comment]
Tests might pass now.

I think this is ready for review.

File: tests/react/address-input.tests.jsx [hunk 1]
```diff
@@ -3,68 +3,63 @@ var TestUtils = require('react/lib/ReactTestUtils');
 var should = require('should');
 var IntlContext = { messages: {'country': 'c', 'address': 'a', city: 'c', postal_code: 'p', 'state_province': 's'} };
 var stubContext = require('react-test-context');
-var {FullAddress} = require('../../components/address-input.jsx');
+var {Address, Code, City} = require('../../components/address-input.jsx');
 
 describe('address-input.jsx', function() {
```
Comment (alicoding) [inline tests/react/address-input.tests.jsx]: just update this test name

**Comment by alicoding** [comment]
r+

**Pull request merged**